expats_static/deposit_uob.js

```diff
@@ -0,0 +1,19 @@
+  var requestOptions = {
+    method: 'GET',
+    redirect: 'follow'
+  };
+
+  fetch("http://localhost:3000/apideposit", requestOptions)
+    .then(response => response.text())
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/Geofflac/DevOps_ShowAndTell/pullRequest?prid=8167659)
```suggestion
    .then((response) => response.text())
```